### PR TITLE
TS: Panel

### PR DESCRIPTION
**Pull request by arnautov-anton**

### CODE REVIEW

File: src/components/StatusUpdateForm.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 
-import Panel from './Panel';
-import PanelHeading from './PanelHeader';
-import PanelFooter from './PanelFooter';
-import PanelContent from './PanelContent';
+import {
+  Panel,
+  PanelContent,
+  PanelHeader as PanelHeading,
```
Comment (mahboubii) [inline src/components/StatusUpdateForm.js:6]: can we skip the rename here
```suggestion
  PanelHeader,
```

File: src/components/Panel.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <div className="raf-panel-content">{children}</div>
+);
+
+export const PanelFooter = ({ children }: PanelFooterProps) => (
+  <div className="raf-panel-footer">{children}</div>
+);
+
+export const PanelHeader = ({ children }: PanelHeaderProps) => (
+  <div className="raf-panel-header">{children}</div>
+);
```
Comment (mahboubii) [inline src/components/Panel.tsx:28]: these 3 don't accept `panelStyle`

Comment (arnautov-anton) [reply src/components/Panel.tsx:28]: It doesn't, there's an `Omit` at line 12.

image

Comment (mahboubii) [reply src/components/Panel.tsx:28]: why not simply using `{ children: ReactNode }`

File: src/index.tsx [hunk 3]
```diff
@@ -49,10 +49,7 @@ export * from './components/DataLabel';
 export { default as Textarea } from './components/Textarea';
 export { default as EmojiPicker } from './components/EmojiPicker';
 export * from './components/Title';
-export { default as Panel } from './components/Panel';
-export { default as PanelHeading } from './components/PanelHeader';
```
Comment (mahboubii) [inline src/index.tsx:53]: I just realized the default export name is `PanelHeading`, we should use PanelHeading everywhere then

File: src/components/Panel.tsx (hunk 2 above)
Comment (arnautov-anton) [reply src/components/Panel.tsx:28]: Just to make it look cooler. 😎 I mean, there's little to no difference (same LoC) and this way it "points" to the `PanelProps` to simulate inheritance of sorts if you know what I mean.

File: src/index.tsx (hunk 3 above)
Comment (arnautov-anton) [reply src/index.tsx:53]: Missed that one, will fix in a bit.

File: src/components/Panel.tsx (hunk 2 above)
Comment (mahboubii) [reply src/components/Panel.tsx]: haha simplicity is usually better but I see your point

**Pull request merged**